src/App.tsx

```diff
@@ -1,24 +1,108 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { UserWarning } from './UserWarning';
+import { createTodo, getTodos, removeTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { Errors, Sort } from './utils/enums';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+// eslint-disable-next-line import/no-cycle
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+export const USER_ID = 10284;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const [sort, setSort] = useState(Sort.All);
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const [Error, setError] = useState<Errors | null>(null);
+
+  const loadTodos = useCallback(async () => {
+    try {
+      const todoList = await getTodos(USER_ID);
+
+      setTodos(todoList);
+    } catch {
+      setError(Errors.Url);
+    }
+  }, []);
+
+  if (Error) {
```

Comment:
Use 'if (error)' instead of 'if (Error)' to check for errors.

Reply:
But this Error is going from the state